fr/integrations/plugin-packs/procedures/hardware-storage-lenovo-iomega-snmp.md

```diff
@@ -0,0 +1,182 @@
+et configurez les paramètres SNMP v3 via la macro SNMPEXTRAOPTIONS.
+
+| Mandatory   | Name                    | Description                                                                                 |
+| :---------- | :---------------------- | :------------------------------------------------------------------------------------------ |
+|             | SNMPEXTRAOPTIONS        | Extra options SNMP de Dell Xseries                                                          |
+
+## FAQ
+
+#### Comment faire le test en ligne de commande et que signifient les principales options ?
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande depuis votre collecteur Centreon avec l'utilisateur _centreon-engine_ :
+
+```bash
+/usr/lib/centreon/plugins/centreon_lenovo_iomega_snmp.pl
+    --plugin=storage::lenovo::iomega::snmp::plugin
+    --mode=cpu
+    --hostname=10.30.2.114
+    --snmp-version='2c'
+    --snmp-community='iomega_ro'
+	--warning-average='60'
+	--critical-average='75'
+    --verbose
+	
+OK: CPU(s) average usage is 15.29 % - CPU '0' usage : 15.29 % 
+| 'total_cpu_avg'=15.29%;0:60;0:75;0;100 'cpu'=15.29%;;;0;100
+```
+
+Cette commande contrôle l'utilisation CPU (```--mode=cpu```) d'un équipement ayant pour adresse *10.30.2.114* (```--hostname=10.30.2.114```) 
+en version *2c* du protocol SNMP (```--snmp-version='2c'```) et avec la communauté *iomega_ro* (```--snmp-community='iomega_ro'```).
+
+Cette commande déclenchera une alarme WARNING si la moyenne du CPU augmente à plus de 60% (```--warning-average='60'```)
+et une alarme CRITICAL si plus de 75% (```--critical-average='75'```).
+Des seuils peuvent être fixés sur toutes les métriques de l'appareil en utilisant la syntaxe "```--warning-*metric* --critical-*metric*```".
+ 
+Toutes les options qui peuvent être utilisées avec ce plugin se trouvent sur la commande ```--help``` :
+
+```bash
+/usr/lib/centreon/plugins/centreon_lenovo_iomega_snmp.pl
+    --plugin=storage::lenovo::iomega::snmp::plugin
+    --mode=cpu \
+    --help
+```
+
+### UNKNOWN: SNMP GET Request : Timeout
+
+Si vous obtenez ce message, cela signifie que vous ne parvenez pas à contacter le serveur Linux sur le port 161, ou alors que la communauté SNMP configurée n'est pas correcte. 
+Il est également possible qu'un firewall bloque le flux.
+
+### UNKNOWN: SNMP GET Request : Cant get a single value.
+
+Si vous rencontrez cette erreur, il est probable que les autorisations données à l'agent SNMP soient trop restreintes. 
+ * L'équipement Lenovo Iomega ne prend pas en charge la MIB utilisée par le Plugin.
+ * L'OID SNMP ciblé ne peut pas être récupéré en raison de privilèges d'équipement insuffisants.
+ 
+Si cela se produit sur le mode Inodes, il est probable que le service SNMP du serveur Linux ne soit pas correctement configuré, il vous faut ajouter la directive 
+ci-dessous dans le fichier de configuration SNMP puis redémarrer le service: 
+```includeAllDisks 10%```
```

Comment:
I don't know if it's very revelant here.

Reply:
I removed this part because, it's necessary